core/src/display_object/movie_clip.rs

```diff
@@ -2308,6 +2308,9 @@ impl<'gc> MovieClipData<'gc> {
     fn stop_audio_stream(&mut self, context: &mut UpdateContext<'_, 'gc, '_>) {
         if let Some(audio_stream) = self.audio_stream.take() {
             context.stop_sound(audio_stream);
+            if context.audio_manager.sounds_count() == 0 {
+                *context.frame_rate = Some(*context.root_frame_rate);
```

Comment:
What happens if the stage framerate was changed with `Stage.frameRate` in AS3, and then streaming sounds end? Does it revert to the root movie's framerate or the one set by AS3?

We probably should write a test for this. All of the relevant code should run on `NullAudioBackend` etc. If you're not sure how to write tests I can walk you through it on Discord or just provide you with one.

Reply:
Next week I will not be able to work on this - don't even know what AS3 is. So could you supply the tests?